pkg/kube/cluster-init.sh

```diff
@@ -295,7 +294,7 @@ check_start_containerd() {
                 # This is very similar to what we do on kvm based eve to start container as a VM.
                 logmsg "Trying to install new external-boot-image"
                 # This import happens once per reboot
-                if ctr -a /run/containerd-user/containerd.sock image import /etc/external-boot-image.tar ${EVE_EXTERNAL_BOOT_IMG}; then
+                if ctr -a /run/containerd-user/containerd.sock image import /etc/external-boot-image.tar docker.io/lfedge/eve-external-boot-image:latest; then
```

Comment:
Looks like you are reverting a change which was done a few weeks ago (which was done to get repeatable builds).

Reply:
Ok

Reply:
@eriknordmark I have a rework of fixing this to a static tag almost ready, can we hold on merging until I push it up?